### PR TITLE
Create Apple object

### PR DESCRIPTION
**Pull request by EmilyCook321**
First commit
1. uploaded apple svg &  introduced🍎 in fixed position
Second commit
2. in elements.js created generateRandomNumber & exported to game.js
3. applied function createGameObject & drawGameObject

### CODE REVIEW

File: src/components/game/game.js [hunk 1]
```diff
@@ -57,7 +57,11 @@ const createPlayer = () => {
 };
 
 const createGameObject = (gameObjectImage, canvas) => {
-  return { img: gameObjectImage, xPos: 20, yPos: 20 };
+  return {
+    img: gameObjectImage,
+    xPos: generateRandomNumber(canvas.width),
+    yPos: generateRandomNumber(canvas.height),
+  };
```
Comment (manuelsanchez2) [inline src/components/game/game.js:64]: lg2m!

**Review by manuelsanchez2 — APPROVED** [review]

File: src/components/game/game.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .game {
-  background-color: #050505;
+  background-color: hotpink;
```
Comment (manuelpapa) [inline src/components/game/game.css:2]: better use only one system (rgb / HEX / etc.)

**Review by manuelpapa — COMMENTED** [review]
Great job!

File: src/components/game/game.css (hunk 2 above)
Comment (EmilyCook321) [reply src/components/game/game.css:2]: good point, thanks!

**Pull request closed without merging**